libs/test-common/src/main/java/org/codice/ddf/test/common/mockito/PrivilegedVerificationMode.java

```diff
@@ -0,0 +1,142 @@
+/**
```

Comment:
🤔 Should we start thinking about having an external test project where we concentrate utilities such as that one and others we have that have nothing to do with DDF directly?